assets/js/components/event/StreamArea.tsx

```diff
@@ -110,18 +110,16 @@ const StreamArea = ({
                   presenterName={presenterName}
                   eventChannel={eventChannel}
                   addMessage={undefined}
+                  setCard={setCard}
                 />
-                {device === "mobile" && (
-                  <>
-                    <MobileRightSidebar setCard={setCard} />
-                    <MobileHlsBar
-                      client={client}
-                      eventTitle={eventTitle}
-                      amIPresenter={amIPresenter}
-                      setMode={setMode}
-                      switchAsking={switchAsking}
-                    />
-                  </>
+                {device == "mobile" && (
```

Comment:
It's the best practice to use triple equals operator

Source: https://www.impressivewebs.com/why-use-triple-equals-javascipt/

Reply:
Yes, of course